chrome-app.js

```diff
@@ -1,7 +1,5 @@
-
-chrome.app.runtime.onLaunched.addListener(function(launchData){
-	
-	var open = function(entry){
+chrome.app.runtime.onLaunched.addListener(function(launch_data) {
```

Comment:
I appreciate you changing the variable name to be consistent even tho it's using a non-standard style 😄👍

Reply:
Haha, yeah I did it for consistency. I'd prefer camelCase throughout, but wanted it to be consistent with your existing code (especially since the repo is gaining popularity, and is likely to have PRs with lots of different coding standards).